misk-hibernate/src/main/kotlin/misk/vitess/StartVitessService.kt

```diff
@@ -269,14 +319,25 @@ class StartVitessService(val config: DataSourceConfig) : AbstractIdleService() {
     val docker: DockerClient = DockerClientBuilder.getInstance()
         .withDockerCmdExecFactory(NettyDockerCmdExecFactory())
         .build()
+    /**
+     * Vitess clusters are cached statically so we create a static instance of ResourceLoaderModule.
+     */
+    private val resourceLoader =
+        Guice.createInjector(ResourceLoaderModule()).getInstance(ResourceLoader::class.java)!!
 
     /**
      * Global cache of running vitess clusters.
```

Comment:
For my todo-list . . . some kind of feature in `@MiskTest` that spans all tests without relying on statics and VM hooks

Reply:
Yeah that would be so awesome!